assets/js/components/surveys/CurrentSurvey.js

```diff
@@ -70,23 +73,31 @@ export default function CurrentSurvey() {
 	const currentQuestion = questions?.find( ( { question_ordinal: questionOrdinal } ) => questionOrdinal === currentQuestionOrdinal );
 
 	const answerQuestion = useCallback( ( answer ) => {
-		sendSurveyEvent( 'question_answered', {
-			// eslint-disable-next-line camelcase
-			question_ordinal: currentQuestion?.question_ordinal,
-			answer,
-		} );
+		if ( ! hasAnsweredQuestion ) {
+			setHasAnsweredQuestion( true );
```

Comment:
Let's return early to reduce nesting:
```suggestion
		if ( hasAnsweredQuestion ) {
			return;
		}
		
		setHasAnsweredQuestion( true );
```

Reply:
That's a nice change :smile: